teflo_init/execute/README.rst

```diff
@@ -0,0 +1,53 @@
+Shell
+=====
+
+This example demonstrates how you can execute tests against test machines
+using teflo.
```

Comment:
I think this is not **example demonstrates how you can execute tests against test machines**, but teflo_init folder?